CHANGELOG.md

```diff
@@ -39,6 +39,10 @@
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#1005](https://github.com/realm/SwiftLint/issues/1005)
 
+* Add more helpful reason strings to TrailingCommaRule.  
+  [Matt Rubin](https://github.com/mattrubin)
+  [#1034](https://github.com/realm/SwiftLint/pull/1034)
```

Comment:
we link to the issue a changelog entry resolves, or none at all. See https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes